testing/autofs/APKBUILD

```diff
@@ -0,0 +1,52 @@
+# Contributor: Alex Laskin <alex@lask.in>
+# Maintainer: Alex Laskin <alex@lask.in>
+pkgname=autofs
+pkgver=5.1.4
+pkgrel=0
+pkgdesc="A kernel-based automounter for Linux"
+url="https://www.kernel.org/pub/linux/daemons/autofs/"
+arch="all"
+license="GPL"
+depends=""
+makedepends="linux-headers automake flex bison rpcgen libtirpc-dev libnfs-dev"
+subpackages="$pkgname-doc $pkgname-openrc"
+source="
+  https://www.kernel.org/pub/linux/daemons/${pkgname}/v5/${pkgname}-${pkgver}.tar.xz
+  autofs.initd
+  autofs.confd
+  10-glibc.patch
+  "
+
+build() {
+  cd "$builddir"
+  ./configure \
+    --prefix=/usr \
+    --sysconfdir=/etc/autofs \
+    --with-mapdir=/etc/autofs \
+    --with-confdir=/etc/conf.d \
+    --without-hesiod \
+		--with-libtirpc \
+    --enable-ignore-busy
+  make -j8
+}
+
+check() {
+  cd "$builddir"
+  daemon/automount -V
```

Comment:
Just showing the version is not enough for a `check` function. It can give a false sense of security because usually programs exit early when showing the version. Not sure if there is something else to put here though.

Reply:
This tool has no its own test suite, so I think package maintainer has no more options to test tool than that. I can try to write extensive tests, but should I?

Reply:
From https://wiki.alpinelinux.org/wiki/APKBUILD_Reference : «Creating a very simple check function, that calls program --version is better than disabling tests completely.» (https://github.com/alpinelinux/aports/pull/2322#discussion_r142543002)